tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -299,11 +299,10 @@
 
   BrowserContextMenu.prototype._listAddAppToAppsItem = function(manifestURL) {
     return new Promise((resolve, reject) => {
-      var _ = navigator.mozL10n.get;
       if (!AppInstallManager.getAppAddedState(manifestURL)) {
         resolve({
           type: BUTTON_TYPE,
-          textRaw: _('add-to-apps'),
+          textL10nId: 'add-to-apps',
```

Comment:
I can workaround it by doing `navigator.mozL10n.formatValue('add-to-apps').then(label => {});` or wait for you to fix the `smart-button`.

What do you prefer?